docs/tools.rst

```diff
@@ -22,92 +22,105 @@ Supported input file formats:
 
 Options:
 
-  -h, --help               Help.
-  -v, --version            Version information only.
+  -h, --help               Display this help and exit.
```

Comment:
Make sure you also modify text in the tool. This is just straight copy&paste from the tool.